amundsen_application/static/js/ducks/tableMetadata/api/helpers.ts

```diff
@@ -6,8 +6,8 @@ import { filterFromObj, sortTagsAlphabetical } from 'ducks/utilMethods';
  * Generates the query string parameters needed for requests that act on a particular table resource.
  */
 export function getTableParams(tableDataObject: TableMetadata | GetTableDataRequest): string {
```

Comment:
nit: We can rename this `getTableQueryParams` for clarity, or any other better name you can think of. Not sure if you feel the same way but looking at this method after some time I found this to be a confusing name